### PR TITLE
Add data samples and requirements to Base features

### PR DESCRIPTION
**Pull request by Sergiodero**
Changes to base features include:

- Adds data samples for each feature
- Adds requirements for each feature
- Fixes link to reference.md
- Adds timepoint as a field associated with stop_times

### CODE REVIEW

File: docs/getting_started/features/base.md [hunk 1]
```diff
@@ -6,51 +6,160 @@ All these features must be implemented together to enable a working GTFS feed.
...
+**Pre-requirements**: Implement with rest of base features
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[agency.txt](/schedule/reference/#agencytxt)|`agency_id`, `agency_name`, `agency_url`, `agency_timezone`, `agency_lang`, `agency_phone`, `agency_fare_url`, `agency_email` |
+|[agency.txt](/documentation/schedule/reference/#agencytxt)|`agency_id`, `agency_name`, `agency_url`, `agency_timezone`, `agency_lang`, `agency_phone`, `agency_fare_url`, `agency_email` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table shows basic agency information provided for this file.
```
Comment (isabelle-dr) [inline docs/getting_started/features/base.md]: This is repetitive on the feature description above. I think we can remove it, given that the sample is self-explanatory.

File: docs/getting_started/features/base.md [hunk 2]
```diff
@@ -6,51 +6,160 @@ All these features must be implemented together to enable a working GTFS feed.
...
+**Pre-requirements**: Implement with rest of base features
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[stops.txt](/schedule/reference/#stopstxt)|`stop_id`, `stop_code`, `stop_name`, `stop_desc`, `stop_lat`, `stop_lon`, `stop_url`, `stop_timezone`, `platform_code` |
+|[stops.txt](/documentation/schedule/reference/#stopstxt)|`stop_id`, `stop_code`, `stop_name`, `stop_desc`, `stop_lat`, `stop_lon`, `stop_url`, `stop_timezone`, `platform_code` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table defines a stop providing basic information elements to identify it and locate the stop.
```
Comment (isabelle-dr) [inline docs/getting_started/features/base.md]: This is repetitive on the feature description above. I think we can remove it, given that the sample is self-explanatory.

**Review by isabelle-dr — CHANGES_REQUESTED** [review]
Same meta comments than in the other PRs

- Line break after the ??? note "Sample Data"
- Replace "the following table" with "the following sample"
- Pre-requirements: maybe "all other Base features"?

**Comment by Sergiodero** [comment]
The meta comments have been solved and the descriptions for the agency and stops data samples has been removed. do you mind taking a final look before merging @isabelle-dr? thanks!

File: docs/getting_started/features/base.md [hunk 3]
```diff
@@ -6,51 +6,177 @@ All these features must be implemented together to enable a working GTFS feed.
...
 
+**Pre-requirements**: 
+
+- All other [Base features](/getting_started/features/base)
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[stop_times.txt](/schedule/reference/#stop_timestxt)|`trip_id`, `arrival_time`, `departure_time`, `stop_id`, `stop_sequence`, `pickup_type`, `drop_off_type`, `` |
-
+|[stop_times.txt](/documentation/schedule/reference/#stop_timestxt)|`trip_id`, `arrival_time`, `departure_time`, `stop_id`, `stop_sequence`, `pickup_type`, `drop_off_type`, `time_point` |
```
Comment (isabelle-dr) [inline docs/getting_started/features/base.md]: it's _timepoint_ not _time_point_

**Review by isabelle-dr — APPROVED** [review]
LGTM!

File: docs/getting_started/features/base.md (hunk 3 above)
Comment (Sergiodero) [reply docs/getting_started/features/base.md]: Fixed!

**Pull request merged**